pom.xml

```diff
@@ -129,6 +129,21 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.33</version>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
```

Comment:
What do we need this dependency for? We don't have it in test-hareess

Reply:
We don't, removed